package-lock.json

```diff
@@ -1,8 +1,881 @@
 {
   "name": "brew-helper",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Please remove this file from your PR, you didn't add any new package, so this updates comes automatically